src/index.js

```diff
@@ -1,11 +1,27 @@
 import Resolver from "@forge/resolver";
+import api, { route } from "@forge/api";
 
 const resolver = new Resolver();
 
-resolver.define("getText", (req) => {
+resolver.define("getText", async (req) => {
+  const response = await api.asApp().requestJira(route`/rest/api/3/search`);
+  const data = await response.json();
+  console.log("number of issues", data.total);
+
   console.log(req);
+  console.log(`Project key ${req.context.extension.project.key}`);
+
+  return `Hello, world! HIPPOS ${req.context.extension.project.key}`;
+});
+
+resolver.define("getUsers", async (req) => {
```

Comment:
forge apis must be called in root folder. Then link response to UI folder in `static/hello-world/src`.